chapter-05/final/money/presentation.go

```diff
@@ -0,0 +1,28 @@
+package money
+
+import "fmt"
+
+// ParseAmount transforms user-provided amount into a handy Number type.
+func ParseAmount(amount string) (Number, error) {
```

Comment:
Expose directly `parseNumber` ?

Reply:
It returns a Number, so yes. Alternatively, the Number only represents amounts, so it could be called `Amount` instead.